platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatCollaborativeActionsTestIT.java

```diff
@@ -0,0 +1,94 @@
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomString;
+import static org.exoplatform.platform.qa.ui.selenium.locator.chat.ChatLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+@Tag("chat")
+@Tag("sniff")
+
+public class ChatCollaborativeActionsTestIT extends Base {
+
+
+    HomePagePlatform     homePagePlatform;
+    ManageLogInOut       manageLogInOut;
+    NavigationToolbar    navigationToolbar;
+    RoomManagement       roomManagement;
+    CalendarHomePage     calendarHomePage;
+    EventManagement      eventManagement;
+    UserAddManagement    userAddManagement;
+
+
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+        homePagePlatform = new HomePagePlatform(this);
+        manageLogInOut= new ManageLogInOut(this);
+        navigationToolbar =new NavigationToolbar(this);
+        roomManagement = new RoomManagement(this);
+        calendarHomePage= new CalendarHomePage(this);
+        eventManagement= new EventManagement(this);
+        userAddManagement= new UserAddManagement(this);
+        manageLogInOut.signInCas(PLFData.DATA_USER1, PLFData.DATA_PASS2);
+
+    }
+    @Test
+    public void test01_AddEventInChatRoom() {
+
+        String room = "room" +    getRandomNumber();
+        String username= "username" + getRandomNumber();
+        String event= "event" + getRandomNumber();
+        String location= "location" + getRandomNumber();
+
+        String password= "123456";
+        String FirstName= "FirstName"+getRandomString();
+        String LastName= "LastName" + getRandomString();
+        String email= username + "@test.com";
+
+
+        navigationToolbar.goToAddUser();
+        userAddManagement.addUser(username,password,email,FirstName,LastName);
+        homePagePlatform.goToChat();
+        switchTo().window(1);
+        roomManagement.addRoom(room,username);
+        ELEMENT_CONTACT_LIST.find(byText(room)).click();
+        ELEMENT_COLLABORATION_ACTIONS.click();
+        ELEMENT_CHAT_ADD_EVENT.click();
+        ELEMENT_ADD_EVENT_WINDOW.waitUntil(Condition.appear, Configuration.timeout);
+        ELEMENT_ADD_EVENT.setValue(event);
+        ELEMENT_CHAT_EVENT_FROM_DATE_.click();
+        $(byClassName("today")).click();
+        ELEMENT_CHAT_EVENT_TO_DATE_.click();
+        $(byClassName("today")).click();
+        ELEMENT_CHAT_EVENT_LOCATION.setValue(location);
+        $(byXpath("//*[@id=\"appComposerForm\"]/div[2]/button")).click();
+        ELEMENT_CONTAINER_LIST_MESSAGES.find(byText(event)).waitUntil(Condition.appear,Configuration.timeout);
+        switchToParentWindow();
+        manageLogInOut.signOut();
+        manageLogInOut.signInCas(username,password);
+        $(byClassName("uiIconStatus")).click();
+        ELEMENT_CHAT_NOTIFICATION.find(byText(event)).should(Condition.appears);
+        homePagePlatform.goToCalendarPage();
```

Comment:
1-add check that the event is added in calendar app
2- delete user at the end of the test